petergreve/week4/ass5/app/stores/message-store.js

```diff
@@ -0,0 +1,83 @@
+var Dispatcher = require('../dispatcher/dispatcher');
+var Constants = require('../constants/constants');
+var BaseStore = require('./base');
+
+var _messages = [
+                    {id: 0, listId: 0, text: "testmessage for list 0", isArchived: false},
+                    {id: 3, listId: 0, text: "testmessage for list 0", isArchived: true},
+                    {id: 1, listId: 1, text: "testmessage for list 1", isArchived: false},
+                    {id: 2 ,listId: 2, text: "testmessage for list 2", isArchived: false}
+                 ];
+
+var MessageStore = Object.assign({}, BaseStore, {
+    getAll: function() {
+        return JSON.parse(JSON.stringify(_messages));
+    },
+    get: function(id) {
+        var message = _messages.find(function(m) { return m.id == id });
+        return JSON.parse(JSON.stringify(message));
+    }
+});
+
+ Dispatcher.register(function(payload){
+
+	switch(payload.type) {
+
+        case Constants.CREATE_MESSAGE:
+            if (payload.text.length < 200) {
```

Comment:
The reason I did it this way is because it is stated in the assignment that no view/components should contain any validation, but maybe I did not understand this correctly